src/scripts/StoreRestApi.js

```diff
@@ -15,8 +15,8 @@ export default class StoreRestApi {
 	createSitemap(sitemap) {
 		return this.axiosInstance
 			.post('/sitemaps/', Sitemap.sitemapFromObj(sitemap).exportSitemap())
-			.then(() => {
-				return sitemap;
+			.then(response => {
+				return Sitemap.sitemapFromObj(response.data.body);
```

Comment:
Returned sitemap as it was in sitemap variable when you don't have response.data.body or sitemap inside is not valid.